cli_tools/common/utils/storage/storage_client.go

```diff
@@ -219,12 +219,12 @@ func (hc *HTTPClient) Get(url string) (resp *http.Response, err error) {
 	return hc.httpClient.Get(url)
 }
 
-// StorageObjectDeleter is responsible for deleting object
-type StorageObjectDeleter struct {
-	sc *StorageClient
+// ObjectDeleter is responsible for deleting object
```

Comment:
By removing the Storage prefix, except by looking at the package name, it's not clear what type of object this is deleting. Update docs to reflect this deals with storage objects. Make sure this is addressed in other files (if any).